sources/engine/Xenko.Engine/Engine/LightComponent.cs

```diff
@@ -37,13 +37,13 @@ public LightComponent()
         {
             Type = new LightDirectional();
             Intensity = 1.0f;
-            Id = Interlocked.Increment(ref LightComponentIds);
+            LightComponentId = Interlocked.Increment(ref globalLightComponentId);
         }
 
         /// <summary>
         /// Internal id used to identify a light component
         /// </summary>
-        internal readonly int Id;
+        internal readonly int LightComponentId;
```

Comment:
Is that really needed? Let's say I have a variable of type `LightComponent` named `lightComponent`. I'll have to write `lightComponent.LightComponentId`. I find it verbose for no reason. `lightComponent.Id` is much easier to read.

Reply:
I think you've missed the reason for the change. `Id` already exists in a base type and is a `Guid`. The rename is to avoid a clash since it's clearly used differently from the base type (as a global counter).

Reply:
This `LightComponent.Id` can be removed.
I was doing a separate warning pass (while adding StyleCop) and just removed that one.

Reply:
@gitfool I missed it obviously (that's what happens when you review online on a mobile phone).
@xen2 OK let's remove it then.